budgetApp1/budgetApp1/UISpecs.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  UISpecs.swift
+//  budgetApp1
+//
+//  Created by Lehlohonolo Nkadimeng on 2024/04/20.
+//
+
+import Foundation
+import UIKit
+
+class UISpecs {
+    let appCustomColor = UIColor(red: 130/255.0, green: 120/255.0, blue: 225/255.0, alpha: 1.0)
+    let appCustomColor2 = UIColor(red: 61/255.0, green: 28/255.0, blue:230/255.0, alpha: 1.0)
```

Comment:
appCustomColoTwo
rather call it primary and secondary colours

Reply:
Good suggestion!